docs/ADOPTION_KMP_PRODUCT_FLAVORS.md

```
@@ -0,0 +1,405 @@
+  gradle/libs.versions.toml
+
+# Modules apply the convention plugin chained through KMPLibraryConventionPlugin etc.
+grep -rln 'apply\("org\.convention\.kmp\.flavors"\)' \
+  build-logic/convention/src/main/kotlin/ | head -5
+```
+
+#### [§5 — Flavors + dimensions](https://github.com/MobileByteLabs/kmp-product-flavors/blob/development/README.md)
+
+This template uses the **flat DSL** style (v2.4+):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)